[OSDOCS-12387]: Updating ARN text for HCP docs (#83843)
openshift/openshift-docs: modules/hcp-aws-create-role-sts-creds.adoc

+
[source,terminal]
----
$ aws sts get-caller-identity --query "Arn" --output text
----
+
.Example output
[source,terminal]
----
arn:aws:iam::1234567890:user/<aws_username>
----
+

> You need the `arn` value from the output to use in the next step.

Reviewer: The way this is written, I'm not sure if the "arn" value is just a piece of this output, or if it's the entire output, inserted into the <arn> replaceable value in the next step.

If it's the former then clarifying which piece to use would be helpful.

If it's the latter maybe something like the following would be clearer?

```suggestion
Use this output as the value for `<arn>` in the next step.
```

Author: Good suggestion!

$ aws sts get-caller-identity --query "Arn"
----
+
.Example output
[source,terminal]
----

> arn:aws:iam::1234567890:user/<aws_username>

Reviewer 2: if we just want to get the arn without double quotations, I think the command should be 
`$ aws sts get-caller-identity --query "Arn" --output text`

e.g.
$ aws sts get-caller-identity --query "Arn"
"arn:aws:iam::***:user/heli"
$ aws sts get-caller-identity --query "Arn" --output text
arn:aws:iam::***:user/heli

Author: Thanks! I added `--output text`.
